deployment/gcp/variables.tf

```diff
@@ -11,7 +11,7 @@ variable "app_service_account" {
 }
 
 variable "environment" {
-  default     = "production"
+  default     = "prod"
```

Comment:
Why rename to short? This is production in other areas. Is this a GCP convention?

Reply:
I thought the instance name length couldn't be bigger than 32 characters, turns out that I was wrong here. I have reverted.